Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -353,7 +353,16 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
       }
 
     let checkoutValues = createCheckoutEvents.values()
-      .map { $0.checkout.id }
+      .map { values in
+        var checkoutId = values.checkout.id
+
+        if let decoded = decodeBase64(checkoutId), let range = decoded.range(of: "Checkout-") {
+          let id = decoded[range.upperBound...]
+          checkoutId = String(id)
+        }
+
+        return checkoutId
+      }
```

Comment:
Any resolution on why this needs to be decoded from GraphQL? At a minimum, this should really have a ``//TODO: `` explaining the hack.

Reply:
Ah yea i left an unfortunate [comment](https://github.com/kickstarter/ios-oss/pull/1999#issuecomment-2020596093) earlier.

Reply:
its silly that they would give us a string, that we only use to send to one of their mutations, and not format it correctly for us